### PR TITLE
Adding an about page. Frame still

### PR DESCRIPTION
**Pull request by moisesm54**
Wire Frame About Us Page

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import ig from '../public/ico-ig.png';
+import tw from '../public/ico-tw.png';
+import fl from '../public/ico-fl.png';
+import tg from '../public/ico-tg.png';
+import { NavBar } from "../components/navbar";
+
+const Home: NextPage = () => {
+    return (
+        <div>
+            <Head>
```
Comment (RichiCoder1) [inline pages/about.tsx:16]: I added this to the _app, so you should no longer need this.

File: pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import { Box, Container, Flex, Heading, VStack, Text, Link, Stack, Center, PopoverAnchor } from '@chakra-ui/react';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from "next/future/image";
+import mail from '../public/ico-mail.png';
+import fb from '../public/ico-fb.png';
+import ig from '../public/ico-ig.png';
+import tw from '../public/ico-tw.png';
```
Comment (RichiCoder1) [inline pages/about.tsx:8]: It looks like you didn't commit these?

**Review by RichiCoder1 — CHANGES_REQUESTED** [review]

**Comment by moisesm54** [comment]
Weird that some of those lines randomly stayed there... They're not on my machine. >:[

**Comment by RichiCoder1** [comment]
> Weird that some of those lines randomly stayed there... They're not on my machine. >:[

I'ma need you to `git` gud.

**Review by RichiCoder1 — APPROVED** [review]

**Pull request merged**